aci/resource_aci_fabricrsoospath.go

```diff
@@ -25,11 +25,14 @@ func resourceAciOutofServiceFabricPath() *schema.Resource {
 		},
 
 		SchemaVersion: 1,
-		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+		Schema: map[string]*schema.Schema{
 			"annotation": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				DefaultFunc: func() (interface{}, error) {
+					return "orchestrator:terraform", nil
+				},
```

Comment:
Same comment as Akini on other file.

Reply:
Added a GetAnnotation function for uniform schema implementation.